<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelCreatorLauncher</class>
 <widget class="QWidget" name="LevelCreatorLauncher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>LevelCreatorLauncher</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="closePushButton">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="launchLevelPushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Launch selected level
in level creator</string>
   </property>
  </widget>
  <widget class="QListWidget" name="levelList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="createLevelPushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Create new level</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newLevelNameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>New level name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
